### PR TITLE
Improves 'Schedule' tag handling

### PR DESCRIPTION
**Pull request by marcesher**
1. Adds the ability to modify the 'Schedule' tag via new `schedule` and `unschedule commands`.

2. Removes a schedule when an instance is stopped, so that our scheduler doesn't start it.

3. Adds the default 8 AM to 6 PM schedule when an instance is started or extended

### CODE REVIEW

**Comment by marcesher** [comment]
replaced @imuchnik with @vccabral as reviewer b/c Irina won't be available to review for several weeks.

**Comment by vccabral** [comment]
LGTM

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "hubot-aws-cfpb",
-  "version": "1.0.8",
+  "version": "1.0.9",
```
Comment (contolini) [inline package.json:3]: Bump the minor version (`1.0.8` > `1.1.0`) because this change adds backward-compatible features.

**Review by contolini — CHANGES_REQUESTED** [review]

**Comment by contolini** [comment]
@marcesher also please add some notes to readme about how to use the scheduling feature

File: scripts/ec2/schedule.coffee [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+# 8 am to 6 pm
+DEFAULT_SCHEDULE_START = 8
+DEFAULT_SCHEDULE_STOP = 18
+
+createInstancesArray = (instance_ids) ->
+    instances = []
+    instance_ids = instance_ids.replace /^\s+|\s+$/g, ""
+    for i in instance_ids.split /\s+/
+      if i and not i.match(/^--/)
```
Comment (contolini) [inline scripts/ec2/schedule.coffee]: This is picking up the time values, you could do something like `if i and not i.match(/^\-\-/) and i.length > 5`.

Comment (contolini) [reply scripts/ec2/schedule.coffee]: Actually just checking if the string starts with `i-` is much simpler: `if i and i.match(/^i-/)`

**Review by contolini — CHANGES_REQUESTED** [review]

**Review by contolini — APPROVED** [review]

**Pull request merged**